src/main/groovy/com/netgrif/application/engine/DevConsole.groovy

```diff
@@ -79,11 +79,11 @@ class DevConsole {
         }
     }
 
-//    @RequestMapping(value = "/login", method = GET)
+//    @GetMapping(value = "/login")
 //    def login() {
```

Comment:
why is this method here? I would recommend removing it and if we will need it in the future, we will find it in git history